### PR TITLE
Adding Webjobs Prefix and consolidation of namespaces

### PR DESCRIPTION
**Pull request by HarmanDhunna**
# Contributing to the Azure SDK

Please see our [CONTRIBUTING.md](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md) if you are not familiar with contributing to this repository or have questions.

For specific information about pull request etiquette and best practices, see [this section](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md#pull-request-etiquette-and-best-practices).

### CODE REVIEW

**Comment by HarmanDhunna** [comment]
Add all name changes in changelog.md

**Comment by hakimms** [comment]
image
@HarmanDhunna Should we add prefix to `EventTriggerMetrics`?

File: sdk/entra/Microsoft.Azure.WebJobs.Extensions.AuthenticationEvents/src/AuthenticationEventResponseHandler.cs [hunk 1]
```diff
@@ -10,8 +10,8 @@
 using System.Text.Json;
 using System.Threading;
 using System.Threading.Tasks;
+
```
[the remark this answers is not in the document]
Comment (HarmanDhunna) [reply sdk/entra/Microsoft.Azure.WebJobs.Extensions.AuthenticationEvents/src/AuthenticationEventResponseHandler.cs:13]: I don't think so. this is a specific class to this NuGet.

File: sdk/entra/Microsoft.Azure.WebJobs.Extensions.AuthenticationEvents/src/TokenIssuanceStart/TokenIssuanceStartResponse.cs [hunk 2]
```diff
@@ -1,10 +1,6 @@
 ﻿// Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
-using Microsoft.Azure.WebJobs.Extensions.AuthenticationEvents.Framework;
-using Microsoft.Azure.WebJobs.Extensions.AuthenticationEvents.Framework.Validators;
-using Microsoft.Azure.WebJobs.Extensions.AuthenticationEvents.TokenIssuanceStart.Actions;
-
 namespace Microsoft.Azure.WebJobs.Extensions.AuthenticationEvents.TokenIssuanceStart
```
[the remark this answers is not in the document]
Comment (HarmanDhunna) [reply sdk/entra/Microsoft.Azure.WebJobs.Extensions.AuthenticationEvents/src/TokenIssuanceStart/TokenIssuanceStartResponse.cs:4]: we would instead be using OnTokenIssuanceStartResponse. Which would line more w/ what we are doing in graph

**Review by robezirg — APPROVED** [review]

**Pull request merged**